widget/embedded/src/pages/LiquiditySourcePage.tsx

```diff
@@ -145,14 +150,28 @@ export function LiquiditySourcePage({
           value={searchedFor}
         />
 
-        <ListContainer>
-          <List
-            type={
-              <ListItemButton title="_" id="_" onClick={() => console.log()} />
-            }
-            items={filteredList}
-          />
-        </ListContainer>
+        {!filteredList.length && !!searchedFor ? (
```

Comment:
if you sync with the latest version you can use the Search component and have the close icon to remove the text.

Reply:
I pulled the last version of alpha, and now is synced.
thanks for your update.